kbdcfg.lua

```diff
@@ -95,6 +119,7 @@ local function factory(args)
     kbdcfg.current               = args.current or 1
     kbdcfg.menu                  = nil
     kbdcfg.type                  = args.type or "tui"
+    kbdcfg.remember_layout          = args.remember_layout or false
```

Comment:
Could you please align this initialisation with others (just remove 3 spaces before '=')?